### PR TITLE
fix(quantic): fixed validation bug with numeric facet

### PR DESCRIPTION
**Pull request by nathanlb**
Introduced a bug in the previous PR. This is the correct fix. It's very nice to have the tests to validate this now 😛

### CODE REVIEW

**Comment by aelfaleh** [comment]
> Introduced a bug in the previous PR. This is the correct fix. It's very nice to have the tests to validate this now 😛

I think the fix for numeric facet, no?

File: packages/quantic/force-app/main/default/lwc/quanticNumericFacet/quanticNumericFacet.js [hunk 1]
```diff
@@ -364,6 +365,14 @@ export default class QuanticNumericFacet extends LightningElement {
     });
   }
 
+  onChangeMin(evt) {
```
Comment (aelfaleh) [inline packages/quantic/force-app/main/default/lwc/quanticNumericFacet/quanticNumericFacet.js:368]: you add those event to html?

**Review by aelfaleh — APPROVED** [review]

**Review by lbergeron — APPROVED** [review]
It looks good given we test coverage for that.

File: packages/quantic/force-app/main/default/lwc/quanticNumericFacet/quanticNumericFacet.js (hunk 1 above)
Comment (nathanlb) [reply packages/quantic/force-app/main/default/lwc/quanticNumericFacet/quanticNumericFacet.js:368]: i had but the commit didnt pick it up 🤦

Comment (aelfaleh) [reply packages/quantic/force-app/main/default/lwc/quanticNumericFacet/quanticNumericFacet.js:368]: I love the title: `fix(quantic): harry potter and the lost changeset` 😆

File: packages/quantic/force-app/main/default/lwc/quanticNumericFacet/quanticNumericFacet.html [hunk 2]
```diff
@@ -6,7 +6,7 @@
     <div class="slds-size_1-of-1">
       <c-quantic-card-container title={label}>
         <lightning-button-icon
-          class={actionButtonCssClasses}
+          class="actionButtonCssClasses"
```
Comment (aelfaleh) [inline packages/quantic/force-app/main/default/lwc/quanticNumericFacet/quanticNumericFacet.html:9]: `{actionButtonCssClasses}`

File: packages/quantic/force-app/main/default/lwc/quanticNumericFacet/quanticNumericFacet.html [hunk 3]
```diff
@@ -25,7 +25,7 @@
           <ul class="slds-has-dividers_around-space slds-var-m-top_medium">
             <template if:true={withInput}>
               <div class="slds-p-horizontal_x-small">
-                <form onsubmit={onApply} class="facet__search-form slds-grid slds-gutters slds-var-p-bottom_small slds-var-p-top_small">
+                <form onsubmit={onApply} class="slds-grid slds-gutters slds-var-p-bottom_small slds-var-p-top_small">
```
Comment (aelfaleh) [inline packages/quantic/force-app/main/default/lwc/quanticNumericFacet/quanticNumericFacet.html:28]: I need this `facet__search-form` for cypress test

File: packages/quantic/force-app/main/default/lwc/quanticNumericFacet/quanticNumericFacet.html (hunk 2 above)
Comment (nathanlb) [reply packages/quantic/force-app/main/default/lwc/quanticNumericFacet/quanticNumericFacet.html:9]: i dont understand where this came from...

File: packages/quantic/force-app/main/default/lwc/quanticNumericFacet/quanticNumericFacet.html (hunk 3 above)
Comment (nathanlb) [reply packages/quantic/force-app/main/default/lwc/quanticNumericFacet/quanticNumericFacet.html]: i must've made a mistake with my merged after reopening this branch. good catch!

**Pull request merged**